### DISCUSSION TITLE
Will Fees (Assert Location and Antenna Changes) if Paid by Wallet be included in 2022?

### DISCUSSION THREAD

**Discussion by SilverknightFL** [davetapley/helium-tax#87] [General]
I started mining this year, so can't retroactively check to a prior year. Once 2022 is up, will it also so the expenses of the hotspot wallet? I've had to change my antenna specs and re-assert location to a move, which was paid for by the wallet's funds.

**Comment by davetapley** [comment]
@SilverknightFL it **shouldn't**, since this tool is only for 'income' (for USA tax purposes per [here](https://www.irs.gov/irb/2014-16_IRB#NOT-2014-21)).

IANAL but I _assume_ that any HNT payment you make from your wallet (be it assert, paying for beer, or donation) is no different to spending USD you've already paid income tax on.

**Comment by NeverHeardThat** [comment]
Thank you Dave for updating the tool to include 2022 transactions!

**NOT FINANCIAL ADVICE - YOU ARE RESPONSIBLE FOR YOURSELF**

In addition the reference Dave linked to in the last post this [IRS FAQ](https://www.irs.gov/individuals/international-taxpayers/frequently-asked-questions-on-virtual-currency-transactions) has necessary information for considering your taxes.  Despite the name, the IRS is not treating virtual currency as currency.  Virtual currency is being treated as traditional investment assets such as stocks and bonds.  The same rules that apply to those traditional asset classes look like they apply to virtual currency.  Think of any virtual currency you mined as a stock you were forced to purchase with your paycheck so you never saw the USD you were paid.  You paid taxes on the USD earned as income and now you own this virtual currency investment asset.

The fees you pay with HNT to assert location, transfer HNT, etc. are denominated in USD.  You will always pay $0.35 to transfer HNT regardless of the price of HNT (unless changed by a future HIP).  You automatically burn/sell the required amount of HNT to equal $0.35 at the current market price when you execute a transaction.  Therefore, this is essentially a sale of HNT and you will have a capital gain or loss depending on if the HNT burned/sold was done so at a profit or loss compared to the cost basis for the specific tokens burned/sold.  Donations would be treated the same as a gift of stock.  The FAQs partially address donations and cost basis.
